tasks/fork/mint-reth.ts

```diff
@@ -0,0 +1,51 @@
+import { task, types } from "hardhat/config";
+import { Address, encodeFunctionData, parseEther } from "viem";
+import {
+    HyperdriveDeployBaseTask,
+    HyperdriveDeployBaseTaskParams,
+} from "../deploy";
+import {
+    MAINNET_RETH_ADDRESS,
+    MAINNET_RETH_WHALE,
+} from "../deploy/lib/constants";
+
+export type MintRETHParams = HyperdriveDeployBaseTaskParams & {
+    address: string;
+    amount: string;
+};
+
+HyperdriveDeployBaseTask(
+    task(
+        "fork:mint-reth",
+        "Mints the specified amount of RETH to the input address",
+    ),
+)
+    .addParam("address", "address to send RETH", undefined, types.string)
+    .addParam("amount", "amount (in ether) to mint", undefined, types.string)
+    .setAction(
+        async (
+            { address, amount }: Required<MintRETHParams>,
+            { viem, artifacts },
+        ) => {
+            let transferData = encodeFunctionData({
+                abi: (await artifacts.readArtifact("ERC20Mintable")).abi,
```

Comment:
This is fine IMO, but it might be clearer to use `IERC20` since we're not using any of the special functionality introduced by `ERC20Mintable`.